### DISCUSSION TITLE
Trying to replicate work done by MatLab waveletscattering  using kymatio

### DISCUSSION THREAD

**Discussion by tameemalaa** [kymatio/kymatio#788] [Q&A] [answered]
I'm sry but I'm trying to replicate some results done by MatLab [waveletscattering ](https://www.mathworks.com/help/wavelet/ref/waveletscattering.html) and in need of some help. 
1- Does the kymatio library output the same features for the same configuration with Matlab waveletscattering?
2- Is there a way to control the invariance scale in the Kymat library? like in Matlab waveletscattering?
3- Is there any path optimization feature analogous to the "optomizePath" feature in the Matlab waveletscattering?
Any help will be highly appreciated.
Thanks in advance

**Comment by OverLordGoldDragon** [comment, marked as answer]
1. Should be fairly close, but won't be identical - best bet is to compare outputs for identical configurations
 2. `T`
 3. No
